kindsys/cue.mod/module.cue

```diff
@@ -0,0 +1 @@
+module: "github.com/grafana/grafana-app-sdk/kindsys"
```

Comment:
if anything is broken that is not exercised by these tests, i suspect the package name above is the cause